crates/matrix-sdk-ui/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "matrix-sdk-ui"
 version = "0.6.0"
 edition = "2021"
+rust-version = { workspace = true }
```

Comment:
Ohhh, that's neat!

Reply:
We already have it pretty much everywhere, not sure why we didn't here.